Podfile

```diff
@@ -91,7 +91,7 @@ def shared_style_pods
 end
 
 def gutenberg_pods
-  gutenberg tag: 'v1.93.0'
+  gutenberg commit: 'a9bafbb3e6ecf1b0fd2532af5a72a5756f5d1690'
```

Comment:
Heads up that the Gutenberg Mobile reference doesn't point to the last commit from https://github.com/wordpress-mobile/gutenberg-mobile/pull/5663: [`3b7427e1f12a6e34f5b5760c29a3db91cca33857`](https://github.com/wordpress-mobile/gutenberg-mobile/pull/5663/commits/3b7427e1f12a6e34f5b5760c29a3db91cca33857)